inc/namespace.php

```diff
@@ -31,7 +31,10 @@
  */
 function bootstrap() {
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_elasticpress', 4 );
-	add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
+	add_action( 'plugins_loaded', function () {
+		// Run healthchecks on indexes after ElasticPress is loaded.
+		add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
```

Comment:
This change doesn't make any sense to me; why would we defer adding a filter?

Reply:
Because ElasticPress isn't loaded yet and it uses ElasticPress code to perform the healthchecks